sky/jobs/controller.py

```diff
@@ -152,6 +152,20 @@ def _run_one_task(self, task_id: int, task: 'sky.Task') -> bool:
         Other exceptions may be raised depending on the backend.
         """
 
+        latest_task_id, status = managed_job_state.get_latest_task_id_status(
+            self._job_id)
+        is_recovery = False
```

Comment:
Can we call this something else? The task may still be running, in which case we don't necessarily need to "recover" - that is, recover from spot preemption.
Maybe we can call this `is_resume` or `is_restore`?